ui/src/components/ListStateContainer.js

```diff
@@ -49,12 +49,11 @@ class ListStateContainer extends React.Component {
    */
   refreshListFromApi = () => {
     // Construct the request URI.
-    var listUri = "/api/clusters?";
+    var listUri = "/api/clusters?includeDeleted=false";
     if (this.state.perUserFilter) {
       var creatorFilter = encodeURIComponent("creator=" + this.props.googleProfile.email);
-      listUri = listUri + creatorFilter + "&";
+      listUri = listUri + "&" + "_labels=" + creatorFilter;
```

Comment:
Minor comment: this could also be represented as 
```
listUri = listUri + '&' + creatorFilter
```
The `_labels` hack was done because Swagger doesn't support arbitrary query string params (e.g. you have to define the keys).

Reply:
per our slack chat, keeping this in the CL with the following reasoning,
1) A fix is needed since the "creator=" portion was previously url encoded.
2) Keeping the _labels parameter matches the swagger definition.